ArchSketchObject.py

```diff
@@ -0,0 +1,1144 @@
+#***************************************************************************	
+#*                                                                         *	
+#*   Copyright (c) 2018 - 2020                                             *	
+#*   Paul Lee <paullee0@gmail.com>                                         *	
```

Comment:
Please merge L4 in to L3 like so: 
`#*   Copyright (c) 2018-2020 Paul Lee <paullee0@gmail.com>                   *`

Reply:
Thanks :)